# Marine Life Message
std_msgs/Header header

# Species name
string species

# Position
geometry_msgs/Point position
